fdbclient/command_runner.go

```diff
@@ -37,9 +40,32 @@ type realCommandRunner struct {
 	log logr.Logger
 }
 
+// getEnvironmentVariablesWithoutBlacklisted returns the current environment variables for the new process with some
+// FDB specific variables filtered out to ensure we don't set any variables that could change the behaviour of fdbcli or
+// the other fdb tools.
+func getEnvironmentVariablesWithoutExcludedFdbEnv() []string {
```

Comment:
Do you want to exclude these variables only when running "status" command? You are doing it for all commands below.